docs/types/value-types.rst

```diff
@@ -206,7 +205,7 @@ Only expressions of type ``address`` can be converted to type ``address payable`
     and ``address payable`` and just use ``address`` everywhere. For example,
     if you are using the :ref:`withdrawal pattern<withdrawal_pattern>`, you can (and should) store the
```

Comment:
I think we just contradicted this paragraph with the example above, right?

The thing I learnt from the example is that you should do the conversion to address payable as early as possible, so that it is more likely that a caller / user of a function sees that the address might receive ether in the future.

Reply:
> I think we just contradicted this paragraph with the example above, right?

Yeah, I agree. I'm not sure why we don't want to have `owner` in the withdrawal example to be a payable address. IMO, we should replace `payable(msg.sender).transfer...` by `owner.transfer...` whenever possible.